Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+
+public enum TransformType: Int {
+    case resetTransforms
+    case transform
+}
+
+public protocol TransformDelegate : AnyObject {
+    
+    var undoButton: UIBarButtonItem! { get set }
+    var redoButton: UIBarButtonItem! { get set }
+    var resetButton: UIBarButtonItem! { get set }
+    
```

Comment:
I think define these three UIBarButtonItems make TransformDelegate limited. User should be able to choose different UI in order to do undo/redo

Reply:
removed